src/components/molecules/About/AboutPage.tsx

```diff
@@ -0,0 +1,19 @@
+import { Button, Typography } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import { Kurt } from "../../../assets/personalInfo/personalInfo"
+
+export function AboutPage (){
+  const navigate = useNavigate()
```

Comment:
👍 I feel like we should use this everywhere.

Reply:
Navigate? Yes. Kind of the whole point of react navigation. Use the hook it gives you.